python/src/sphericart/spherical_harmonics.py

```diff
@@ -206,3 +206,102 @@ def compute_with_gradients(self, xyz: np.ndarray) -> Tuple[np.ndarray, np.ndarra
             )
 
         return sph, dsph
+
+    def compute_with_hessians(
```

Comment:
Michele and I talked about this, and, although it sounds wrong, it is correct mathematically because each Ylm is a function with its own gradient and Hessian, so it makes sense to use `gradients` and `Hessians`.

Reply:
ok, fair enough